style.css

```diff
@@ -0,0 +1,66 @@
+:root{
+    --primary-color: #DAA06D;
+}
+
+*{
+    box-sizing: border-box;
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
+body{
+    margin: 0;
+    padding: 0;
+}
+/* avatar style */
+.first-avatar{
```

Comment:
You can rename the classes according to their shapes(circle or square) or size(small,medium,large). Naming it according to numbers might look little confusing to the ones who intend to use your components.